silasdk/message.py

```diff
@@ -95,8 +96,7 @@ def createMessage(self, payload, msg_type):
         pass
 
     inpt = cull_null_values(inpt, payload)
-    if (self.debug):
-        print(inpt)
+    logger.debug(inpt)
```

Comment:
In order to respect `self.debug` regardless of the configured logging level, you should get / optionally set / reset the logging levels:
```
log_level = logger.getEffectiveLevel()
if self.debug:
    logger.setLevel(logging.DEBUG)
logger.debug(inpt)
logger.setLevel(log_level)
```

Reply:
It's best practice to set log levels globally regardless of what a specific library's mode is. 

Also, just to clarify: I don't know the reasons behind logging here, I'm just replacing the existing `print` statements.

Reply:
> It's best practice to set log levels globally regardless of what a specific library's mode is.

Unless you want debug logging from just a single module or class, and don't want to see the heinous amount of debug logging that boto3 or other misc packages can spew at times :-)